content/c-sharp/concepts/variables/variables.md

```diff
@@ -12,31 +12,82 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-A **variable** is a storage location in the computer’s memory that is used to save, retrieve, and manipulate data.
+**Variables** are used to store and manipulate data. In C#, each variable has a type that determines the values it can store.
+
+## Types
+
+**C#** is a type-safe language, and has different types. The following are some examples:
+
+| Name     | Description                                                                                                                       |
+| -------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| `int`    | represents a 32-bit signed integer. It can store whole numbers from `-2,147,483,648` to `2,147,483,647`.                          |
+| `double` | represents a 64-bit floating-point number. It can store both whole numbers and decimals from `±5.0 × 10^−324` to `±1.7 × 10^308`. |
+| `char`   | represents a single 16-bit Unicode character. It is used to store a single letter, digit, or symbol.                              |
+| `string` | represents a sequence of characters and is used to store text. It is immutable, meaning that once created, it cannot be changed.  |
+| `bool`   | represents a Boolean value, which can be either `true` or `false`. Used for conditional statements and logical operations.        |
```

Comment:
These are types of data types not variables so add the ones mentioned below 

Local variables
Instance variables or Non – Static Variables
Static Variables or Class Variables
Constant Variables
Readonly Variables